core/services/vrf/integration_helpers_test.go

```diff
@@ -190,7 +190,7 @@ func testSingleConsumerNeedsBHS(
 	keyHash := vrfJobs[0].VRFSpec.PublicKey.MustHash()
 
 	_ = createAndStartBHSJob(
-		t, vrfKey.Address.String(), app, uni.bhsContractAddress.String(), "",
+		t, []string{vrfKey.Address.String()}, app, uni.bhsContractAddress.String(), "",
```

Comment:
Lets add a test that tests BHS stores from multiple from addresses to ensure functionality is as expected.

Reply:
Sounds good. added